pkg/common/plugin/k8s/apiserver/client_test.go

```diff
@@ -117,7 +114,7 @@ oIrPuyjOmscrC627wX3LGUHwPKtNArBT8lKFfda1B1BqAk0q1/ui/A==
 )
 
 const (
-	testToken = "TEST-TOKEN"
+	_testToken = "TEST-TOKEN"
```

Comment:
nit: SPIRE doesn't use this convention for unexported package-level vars

Reply:
Perhaps we should reference a specific style guide in the CONTRIBUTING.md. This convention comes directly from https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_

Reply:
A style guide has been something we've talked about and wanted to introduce before, it has just never been prioritized. I've opened an issue to track this (#2377). Thanks for bringing it up.